### ISSUE TITLE
in PE what does the cone refer to and can you elaborate how its calculated ?

### ISSUE THREAD

**Issue by ETbaty** [exiawsh/StreamPETR#162]
more specifically line 421 in streapetr_head:

cone = torch.cat([intrinsic, coords3d[..., -3:], coords3d[..., -90:-87]], dim=-1)

???
